### PR TITLE
Sessions support for Portals

### PR DESCRIPTION
**Pull request by SanjoDeundiak**

### CODE REVIEW

File: implementations/rust/ockam/ockam/src/lib.rs [hunk 1]
```diff
@@ -129,4 +129,7 @@ pub mod authenticated_storage {
 }
 
 #[cfg(feature = "ockam_transport_tcp")]
-pub use ockam_transport_tcp::{TcpConnectionTrustOptions, TcpListenerTrustOptions, TcpTransport};
+pub use ockam_transport_tcp::{
```
Comment (SanjoDeundiak) [inline implementations/rust/ockam/ockam/src/lib.rs:132]: @etorreborre do you think we could create submodule `ockam::tcp`?

Comment (etorreborre) [reply implementations/rust/ockam/ockam/src/lib.rs:132]: For consistency that would be great!

File: implementations/rust/ockam/ockam_identity/src/channel/trust_options.rs [hunk 2]
```diff
@@ -102,7 +102,7 @@ impl Default for SecureChannelListenerTrustOptions {
 }
 
 impl SecureChannelListenerTrustOptions {
-    /// Constructor
+    /// Constructor without Consumer and Producer sessions, with [`TrustEveryonePolicy
```
Comment (etorreborre) [inline implementations/rust/ockam/ockam_identity/src/channel/trust_options.rs]: The end of line is missing here

File: implementations/rust/ockam/ockam_transport_tcp/src/transport/portals.rs [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    /// Stop outlet at addr
+    /// ```rust
+    /// use ockam_transport_tcp::TcpTransport;
+    /// # use ockam_node::Context;
+    /// # use ockam_core::{AllowAll, Result};
+    /// # async fn test(ctx: Context) -> Result<()> {
+    /// const TARGET_PEER: &str = "127.0.0.1:5000";
+    ///
+    /// let tcp = TcpTransport::create(&ctx).await?;
+    /// tcp.create_outlet("outlet", TARGET_PEER, AllowAll).await?;
```
Comment (etorreborre) [inline implementations/rust/ockam/ockam_transport_tcp/src/transport/portals.rs]: That comment needs to be fixed

**Review by etorreborre — DISMISSED** [review]
🏎️ that's quite straightforward now :-)

**Review by etorreborre — APPROVED** [review]

**Pull request merged**